tests/conftest.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, sessionmaker
+
+from app.database.models import Event, Invitation, User, Base
+
```

Comment:
Consider: create a separate `db_entities` conftest file. 
As far as I know, this is how you reference the other files: 
`pytest_plugins = ['module1', 'module2']`
([stackoverflow answer](https://stackoverflow.com/questions/27064004/splitting-a-conftest-py-file-into-several-smaller-conftest-like-parts))

Reply:
nice ;)